targets/simple_switch_grpc/README.md

```diff
@@ -3,7 +3,7 @@
 This is an alternative version of the simple_switch target, which does not use
 the Thrift runtime server for table programming (unless required, see below).
 Instead it starts a gRPC server which implements
-[p4untime.proto](https://github.com/p4lang/PI/blob/master/proto/p4/p4runtime.proto).
+[p4untime.proto](https://github.com/p4lang/p4runtime/blob/main/proto/p4/v1/p4runtime.proto).
```

Comment:
All of the other changes appear like `master`->`main` branch renaming, which is good.

This one also changes the target repo of the link.  Is that what is desired?

Reply:
Hi Andy, yes, this change is in a separate git commit: https://github.com/p4lang/behavioral-model/pull/1064/commits/426364f4d90efc83bed34d06dedd22c25c9efb31.